docs/src/main/resources/content/_using/replication-limitations.adoc

```diff
@@ -31,3 +31,7 @@ result in the replication of records from the Connected Sources in addition to a
 
 Derived resources, from products such as NITFs, will not be replicated. New derived resources will,
 however, be generated by the receiving ${platform}.
+
+=== Confluence Sources
+
+Metadata received from a confluence source cannot be replicated.
```

Comment:
✏️ Should we wrap this in a note?
```
[NOTE]
====
Metadata received from a confluence source cannot be replicated.
====
```

Reply:
Might mention that any source that generates metacards on the fly (like Confluence) will not work with replication. 
Also should capitalize Confluence

Reply:
I know that it's a small addition but is there any other reason I should make it a note? I think this is a good place for it but it seems odd to have a section with just a note and no other writing.

@clockard Could you explain in more detail how confluence generates metacards and why we can't replicate them? That should help me make the documentation more clear.